xivo/mallow/tests/test_fields.py

```diff
@@ -21,13 +33,14 @@ class AllFieldsSchema(Schema):
     dict_ = fields.Dict()
     email = fields.Email()
     list_ = fields.List(fields.String())
-    nested = fields.Nested('TestSchema')
+    nested = fields.Nested(lambda: TestSchema())
```

Comment:
Why using lambda? According the changelog the "String" usage is not deprecated
https://marshmallow.readthedocs.io/en/stable/changelog.html#id21